.eslintrc

```diff
@@ -11,5 +11,31 @@
   "rules": {
     "no-console": 1, // Means warning
     "prettier/prettier": 2 // Means error
-  }
+  },
+  "overrides": [
+    {
+      "files": ["**/*.ts"],
+      "env": { "browser": true, "es6": true, "node": true },
+      "extends": [
+        "eslint:recommended",
+        "plugin:@typescript-eslint/eslint-recommended",
+        "plugin:@typescript-eslint/recommended"
+      ],
+      "globals": { "Atomics": "readonly", "SharedArrayBuffer": "readonly" },
+      "parser": "@typescript-eslint/parser",
+      "parserOptions": {
+        "ecmaVersion": 2018,
+        "sourceType": "module",
+        "project": "./tsconfig.json"
+      },
+      "plugins": ["@typescript-eslint"],
+      "rules": {
```

Comment:
These styling rules really belong in prettier, otherwise they conflict.

Reply:
Rules removed